### PR TITLE
Duckbot corrects the spelling of that guy

### PR DESCRIPTION
**Pull request by jameshughes89**
##### Summary

Although very rare, we must ensure correctness when someone mentions that dude from Fox News' name. Duckbot will automatically correct us whenever someone mentions him with his proper name. 

Effectively copy\pasted the bitcoin cog and tests. 

##### Checklist

- [x] this is a source code change
  - [x] run `format` in the repository root
  - [x] run `pytest` in the repository root
  - [ ] link to issue being fixed using a [_fixes keyword_](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue), if such an issue exists
  - [x] update the wiki documentation if necessary
- [ ] or, this is **not** a source code change

### CODE REVIEW

File: duckbot/cogs/corrections/tarleson.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+        self.bot = bot
+
+    @commands.Cog.listener("on_message")
+    async def correct_tarleson(self, message):
+        """Corrections for tarleson"""
+
+        if message.author == self.bot.user:
+            return
+
+        if "tucker carleson" in message.content.lower():
```
Comment (twentylemon) [inline duckbot/cogs/corrections/tarleson.py]: maybe also look for "tucker carlson" which is how I assumed his name was spelled

**Review by twentylemon — COMMENTED** [review]
Yo dawg, also update the wiki pages.

File: duckbot/cogs/corrections/tarleson.py (hunk 1 above)
Comment (jameshughes89) [reply duckbot/cogs/corrections/tarleson.py]: I agree that looking for the actual correct spelling is a good move.

**Comment by jameshughes89** [comment]
I was considering adding "**comrade** cucker tarleson". 

If I see +1 I will update accordingly.

**Comment by jameshughes89** [comment]
> Yo dawg, also update the wiki pages.

:+1:

**Review by twentylemon — APPROVED** [review]
image

**Pull request merged**